app/jsx/pages/SearchPage.jsx

```diff
@@ -455,12 +462,17 @@ class SearchPage extends PageBase {
                 <h2 className="o-columnbox1__heading">
                   Scholarly Works ({data.count + " results" + (data.count > 10000 ? ", showing first 10000" : "")})</h2>
               </header>
+            {(data.count > 10) &&
```

Comment:
Good point that this should be checked! However, instead of a constant 10, shouldn't it check against the currently selected number of results per page?